pages/04.tutorials/26.sharing-volumes-between-pods/default.en.md

```diff
@@ -13,7 +13,7 @@ taxonomy:
 ## Overview
 
 This document describes how to setup a demo deployment with a [persistent volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/). Normally, when using a ReadWriteOnce Volume, you cannot mount this volume onto another pod.
-This is normally only possible with [RWX (ReadWriteMany)](https://docs.syseleven.de/metakube/en/tutorials/read-write-many-volumes).
+This is normally only possible with RWX (ReadWriteMany), which is currently not supported by MetaKube. For some use cases the following can be a valuable workaround to achive similar results.
```

Comment:
```suggestion
This is normally only possible with RWX (ReadWriteMany), which is currently not supported by MetaKube. For some use cases the following can be a valuable workaround to achieve similar results.
```

Reply:
I'd replace the old link with one to to an external documentation. Probably this one https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes

Reply:
https://github.com/syseleven/grav-docs-metakube/pull/134/files